packaging/windows/split.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+const chunkSize = 1024 * 1024 * 1024 // 1GiB chunk size
+
+func main() {
+	if len(os.Args) != 2 {
+		log.Fatal("Split takes only one argument (the file to split)")
+	}
+
+	if err := split(os.Args[1]); err != nil {
+		log.Fatal(err.Error())
+	}
+}
+
+func split(filePath string) error {
+	bundle, err := os.Open(filePath)
+	if err != nil {
+		return err
+	}
+	defer bundle.Close()
+
+	bundleFileStat, err := bundle.Stat()
+	if err != nil {
+		return err
+	}
+
+	for i := 0; ; i++ {
+		partFileName := fmt.Sprintf("%s.%d", bundleFileStat.Name(), i)
+
+		err := func() error {
+			partFile, err := os.Create(filepath.Join(filepath.Dir(filePath), partFileName))
+			if err != nil {
+				return err
+			}
+			defer partFile.Close()
```

Comment:
I would just remove the anonymous function, and let all these `defer` happen at the end of `split`, I checked that this will do the right thing, and close all the files which were opened. Alternatively, I would use a separate function with a good name which would improve readability by making the functions shorter, and by adding some english description of what the loop is doing.
Here I feel the anonymous function is making the code more complicated to read, and could easily be avoided.